configuration/editors.md

```diff
@@ -19,25 +19,24 @@ This documentation page can give you some guidance in choosing the right one for
 ## openHAB VS Code Extension
 
 openHAB VS Code is an extension for the [Visual Studio Code](https://code.visualstudio.com) editor.
+You can find it in the [Microsoft Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=openhab.openhab).
 
-  ![openHAB VS Code Extension alternative installation](images/vscode_demo.gif)
+  ![openHAB VS Code Extension demo](images/vscode_demo.gif)
 
 ### Installation
 
-1. Download the latest release from [openhab-vscode](https://github.com/openhab/openhab-vscode/releases)
-2. Install the extension with one of the following methods
-    - Open command line in the same folder you downloaded the extension
-      Type the following command:
-      ```shell
-        code --install-extension openhab-0.0.2.vsix
-      ```
-    - Go to the extensions area and install the extension like showed in the screenshot:
-      ![openHAB VS Code Extension alternative installation](images/vscode_alternative_install.png)
-
-Currently openhab-vscode isn't yet listed in [Microsoft Visual Studio Marketplace](https://marketplace.visualstudio.com/vscode), but the contributors are working on it already.
+1. Open the extension sidebar. ![openHAB VS Code Extension alternative installation](images/vscode_extensiontab_icon.png)
```

Comment:
Should the step 0 be "Install VS Code if not yet available"?

Reply:
`0. Install [Visual Studio Code](https://code.visualstudio.com/Download) if not yet available` ;-)